foundation/foundation-upgrade/src/main/resources/migrations.json

```diff
@@ -109,6 +118,15 @@
             ]
           }
         ]
+      },
+      {
+        "name": "data-lineage-package-import-migration",
+        "implementation": "com.boozallen.aissemble.upgrade.migration.v1_7_0.DataLineageAiopsReferencesMigration",
+        "fileSets": [
+          {
+            "includes": ["*-pipelines/*/src/*/java/com/boozallen/cdi/*.java"]
```

Comment:
I: In Maven/Ant fileglob syntax, `**` matches multiple directories.  I already recommended a complete change to our glob in the first migration instance, but you wouldn't have needed this if you had made that includes `*-pipelines/*/src/*/java/**/cdi/*.java`